gammapy/data/event_list.py

```diff
@@ -386,13 +386,114 @@ def _get_lon_lat(self, header):
     def peek(self):
         """Summary plots."""
         import matplotlib.pyplot as plt
-        fig, axes = plt.subplots(nrows=1, ncols=3, figsize=(20, 8))
+        fig, axes = plt.subplots(nrows=1, ncols=2, figsize=(20, 8))
         self.plot_image(ax=axes[0])
-        self.plot_energy_dependence(ax=axes[1])
-        self.plot_offset_dependence(ax=axes[2])
+        self.plot_time_map(ax=axes[1])
+
+        #log-log scale for time map
+        xlims = axes[1].set_xlim()
+        ylims = axes[1].set_ylim()
+        axes[1].set_xlim(1e-3, xlims[1])
+        axes[1].set_ylim(1e-3, ylims[1])
+        axes[1].loglog()
+        # TODO: self.plot_energy_dependence(ax=axes[x])
+        # TODO: self.plot_offset_dependence(ax=axes[x])
         plt.tight_layout()
         plt.show()
 
+    def plot_image(self, ax= None, number_bins=50):
+        """Plot the counts as a function of x and y camera coordinate."""
+        import matplotlib.pyplot as plt
+        ax = plt.gca() if ax is None else ax
+
+        max_x = max((self[:]['DETX']))
+        min_x = min((self[:]['DETX']))
+        max_y = max((self[:]['DETY']))
+        min_y = min((self[:]['DETY']))
+
+        x_edges = np.linspace(min_x, max_x, number_bins)
+        y_edges = np.linspace(min_y, max_y, number_bins)
+
+        count_image, x_edges, y_edges = np.histogram2d(self[:]['DETY'], self[:]['DETX'], bins=(x_edges, y_edges))
+
+        ax.set_title('# Photons')
+
+        ax.imshow(count_image, interpolation='nearest', origin='low',
+                  extent=[x_edges[0], x_edges[-1], y_edges[0], y_edges[-1]])
+
+    def plot_energy_dependence(self, ax = None):
+        import matplotlib.pyplot as plt
+        ax = plt.gca() if ax is None else ax
+
+    def plot_offset_dependence(self, ax = None):
```

Comment:
Same here ... an offset histogram would be nice to have.

An energy-offset histogram will come soon, please don't try to implement it in this PR:
https://github.com/gammapy/gammapy/pull/383